test/cross-browser/reference.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <!-- meta -->
+        <meta charset="utf-8" />
+        <title>rc-socket reference implementation</title>
+
+        <!-- external script -->
+        <script src="../../dist/rc-socket.js"></script>
+
+        <!-- styles -->
+        <!--<link rel="stylesheet" href="../node_modules/easy-build/node_modules/mocha/mocha.css" />-->
+    </head>
+
+    <!--<script src="http://cdnjs.cloudflare.com/ajax/libs/loglevel/1.4.0/loglevel.min.js"></script>-->
+
+    <!-- content -->
+    <div id="workboard" style="position: absolute;"></div>
+    <div id="mocha">
+        <input type="button" onclick="createRcSocket();" value="StartSocket"/>
+    </div>
+
+    <!-- traces -->
+    <script>
+        var childWin;
+
+        var createRcSocket = function() {
+            var child = window.open('', 'childLogger');
+            childWin = child.window;
+
+            if (!childWin.clientLogger) {
+                childWin.clientLogger = {
+                    buffer: new Array(),
+
+                    debug: function() {
+                        childWin.clientLogger.buffer.push(
+                            arguments['0'] + ' ' + arguments['1'] + ' ' + arguments['2']
```

Comment:
For now this seems ok.

We talked the other day and I just wanted to provide a little more insight into the arguments object:
 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments

Usually within an application we utilize a lib like lodash/underscore and we could transform the code:

```
childWin.clientLogger.buffer.push(_.toArray(arguments).join(' '));
```